api/src/pcapi/core/users/external/user_automations.py

```diff
@@ -126,23 +126,22 @@ def get_email_for_inactive_user_since_thirty_days() -> Iterable[str]:
 
     return (
         email
-        for email, in get_young_users_emails_query().filter(func.date(User.lastConnectionDate) <= date_30_days_ago)
+        for email, in get_young_users_emails_query().filter(func.date(User.lastConnectionDate) == date_30_days_ago)
```

Comment:
A partir de ce moment, une itération qui fail (erreur externe par exemple) n'est pas automatiquement rattrapée par la suivante n'est ce pas ?
Qu'est ce que tu pense de faire en sorte que la date_30_days_ago soit un paramètre de ton cron, comme ça on pourra relancer à la main le jour suivant en donnant la bonne date en paramètre ?
Par exemple faire
`flask command push_to_sendinblue_inactive_user --date=25/12/2021`

Reply:
Peut être mettre ce changement dans un autre commit, vu que c'est un deuxième changement fonctionnel ?

Reply:
Sinon, on peut prendre par exemple entre 30 et 45 jours, ça fait une fenêtre glissante

Reply:
Pourquoi pas oui

Reply:
Fait.